BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -751,6 +751,20 @@ void clear(FaceGraph& g)
   CGAL_postcondition(num_faces(g) == 0);
 }
 
+
+template <typename FaceGraph>
+typename boost::graph_traits<FaceGraph>::degree_size_type
+face_degree(typename boost::graph_traits<FaceGraph>::halfedge_descriptor hd, const FaceGraph& g)
+{
+  typename boost::graph_traits<FaceGraph>::degree_size_type res = 0;
+  typename boost::graph_traits<FaceGraph>::halfedge_descriptor done = hd;
+  do {
+    ++res;
+    hd = next(hd,g);
+  }while(! (hd == done));
+  return res;
+}
```

Comment:
And we cannot call `degree(face(hd,g), g)` as the face might be the  `null_face()`.  

We will remove the new function, and in the code write  `boost::distance(CGAL::halfedges_around_face(hd,g))`.